.github/workflows/release-with-jec.yml

```diff
@@ -38,384 +43,435 @@ jobs:
       integration_name: ${{  steps.gather_params.outputs.INTEGRATION_NAME }}
       integration_version: ${{  steps.gather_params.outputs.INTEGRATION_VERSION }}
     steps:
-    - name: Check out code into the Go module directory
```

Comment:
can you please disable the code formatter in your local?

Reply:
done